Double_index/345-Reverse_Vowels_of_a_String.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+'''
+Write a function that takes a string as input and reverse only the vowels of a string.
+
+Example 1:
+
+Input: "hello"
+Output: "holle"
+Example 2:
+
+Input: "leetcode"
+Output: "leotcede"
+Note:
+The vowels does not include the letter "y".
+'''
+
+class Solution(object):
+    # Time complexity: O(n)
+    # Space complexity: O(1)
+    def reverseVowels(self, s):
+        """
+        :type s: str
+        :rtype: str
+        """
+        vowels = ['a', 'e', 'i', 'o', 'u', 'A','E','I','O','U']
+
+        leftPointer = 0
+        rightPointer = len(s) - 1
+        s = list(s)
+
+        while leftPointer < rightPointer:
+            if s[leftPointer] not in vowels:
```

Comment:
This lookup and the one below can be further optimized.

Reply:
how can they be optimized?

Reply:
https://stackoverflow.com/questions/513882/python-list-vs-dict-for-look-up-table

Reply:
I've converted list to set